client/src/docs-ui/search-bar/search-bar.style.ts

```diff
@@ -0,0 +1,93 @@
+import {MQLaptop} from "../../amplify-ui/styles/media";
+import {css} from "emotion";
+
+// we use !important to override webkit's imposed border-radius
+export const searchStyle = css`
+  width: 100%;
+
+  ${MQLaptop} {
+    width: initial;
+    margin: 0 1rem;
+  }
+
+  .algolia-autocomplete .ds-dropdown-menu {
+    min-width: initial;
+
+    ${MQLaptop} {
+      min-width: 500px;
+    }
+  }
+
+  > div {
+    width: 100%;
+    padding: 0 1rem;
+
+    > div {
+      display: flex;
+      flex: 1;
+      flex-direction: row;
+      align-items: center;
+      position: relative;
+
+      .algolia-autocomplete {
+        display: block;
+        width: 100%;
+      }
+
+      .algolia-autocomplete .ds-dropdown-menu * {
+        white-space: normal;
+      }
+
+      .algolia-autocomplete
+        .algolia-docsearch-suggestion--subcategory-column-text {
+        white-space: normal;
+      }
+
+      .algolia-autocomplete
+        .algolia-docsearch-suggestion--content
+        > .algolia-docsearch-suggestion--text
+        > .algolia-docsearch-suggestion--highlight {
+        box-shadow: inset 0 -2px 0 0 var(--color-orange-hv);
+      }
+
+      .algolia-autocomplete .algolia-docsearch-suggestion--title {
+        font-weight: bold;
+        color: black;
+      }
+
+      .algolia-autocomplete .algolia-docsearch-suggestion--text {
+        font-size: 0.8rem;
+        color: gray;
+      }
+
+      .ds-cursor .algolia-docsearch-suggestion--content {
+        background-color: var(--bg-color-hover) !important;
```

Comment:
Is there another option we can use in order to avoid using `!important`?

Reply:
I commented above that style. Algolia's styles were a pain to override. Time-boxed it and moved on. I urge you to try... although this one's a tricky beast! It's tough––in this case––to get a more specific selector, which accounts for conflict with algolia-injected styles.

Reply:
& yea––will make that change